### ISSUE TITLE
Several buttons in web app not labelled correctly

### ISSUE THREAD

**Issue by limitedmage** [sourcegraph/sourcegraph#12914]
The following buttons/links in the main code view are not labelled appropriately:

- Sourcegraph logo (reads image URL)
- Hamburger menu [edit: command palette icon] (reads "button") 
- User profile dropdown (reads image URL)
- Breadcrumbs bar dropdowns (reads "button")
- Copy to clipboard button at the end of the breadcrumbs bar (reads "button"; note button is hidden and will only show on mouse hover, but VoiceOver can still focus it)
- Raw file download button (reads file name)
- Show sidebar button (reads "button")

![Screen Shot 2020-08-11 at 8 52 15 AM](https://user-images.githubusercontent.com/206864/89920262-0ce12f00-dbb1-11ea-9d61-a19581620e73.png)

**Comment by jasongornall** [comment]
closing this as these are now duplicated from the user stories from accessibility (many of these have already been addressed)
